### DISCUSSION TITLE
blog/setting-up-ubuntu-development-environment-with-docker-on-macos

### DISCUSSION THREAD

**Discussion by giscus** [shenlu89/shenlu.me#49] [Announcements]
# blog/setting-up-ubuntu-development-environment-with-docker-on-macos

Setting up Ubuntu Development Environment with Docker on MacOS

https://www.shenlu.me/blog/setting-up-ubuntu-development-environment-with-docker-on-macos

**Comment by choneas** [comment]
Hello I am fom ZhiHu with your Next.js blog build article. I am also learning and building a Next.js blog.

I cloned your site's repo and I am applying it to my site. It's **perfect** and easy to understand! I love it 😋

**Reply by shenlu89** [reply]
Thank you for telling me, I’m glad I could help you.

**Comment by morristech** [comment]
You docker file won't run as it currently is add unzip after tmux in Dockerfile

**Reply by shenlu89** [reply]
Good catch, thanks! Fixed — added unzip to the apt install step before running the Bun installer.
